rockcraft.yaml

```diff
@@ -9,7 +9,12 @@ description: |
 license: GPL-3.0
```

Comment:
could there be a problem with your branch? I see the license is changed in both commits, but overall in PR's "Files changed" it's not modified

Reply:
Ok this is very weird, I force pushed my commits again and it seems fixed now.